december-2017/php-version.md

```diff
@@ -11,3 +11,5 @@ PHP 5.2 has served us well but has not received any performance or security upda
 In the interest of ensuring MyBB remains up to date with current best practices and to leverage the new features and optimisations generally found in new versions of PHP, we propose that the minimum PHP version requirement be increased with the release of MyBB 1.10 - from PHP 5.2 to either 5.6 (which has just over a year until it reaches EOL on December 31, 2018) or 7.0 (which has around two years). MyBB 1.8 would continue to support the PHP versions that it currently supports, should users be unable to update the versions of software running on their servers.
 
 MyBB 1.10 development is underway (some PRs have entered the review stage and suggestions are being taken in the [1.10 Suggestions and Feedback](https://community.mybb.com/forum-199.html) forum). We are really keen to gather as much feedback as possible and to learn from our userbase what their concerns might be when met by a change as substantial as this.
+
+The 1.8 branch, while carrying structural concepts of over 12 years (which have passed recently [since MyBB 1.0.0](https://mybb.com/versions/)), is expected to remain compatible with PHP 5.2 as well as with more recent versions to prevent problems resulting from raised requirements in minor releases as the work on modern branches continues.
```

Comment:
> which have passed recenty sunce MyBB 1.0.0

This sounds weird, but I'm not sure what would make more sense.

Reply:
Something like this a bit better?

> The MyBB 1.x series has continued to use the same structural concepts from [MyBB 1.0](https://mybb.com/versions/) for over 12 years. While MyBB 1.10 will have a different compatibility, MyBB 1.8 will remain compatible with PHP 5.2 as well as newer versions of PHP.
<details><summary></summary><p>

```markdown
The MyBB 1.x series has continued to use the same structural concepts from [MyBB 1.0](https://mybb.com/versions/) for over 12 years. While MyBB 1.10 will have a different compatibility, MyBB 1.8 will remain compatible with PHP 5.2 as well as newer versions of PHP.
```

</p>
</details>